src/components/mermaidInputs/InputWithLabelAndValidation/InputWithLabelAndValidation.js

```diff
@@ -21,13 +35,18 @@ const InputWithLabelAndValidation = ({
   unit,
   validationMessages,
   validationType,
+  addCheckbox,
+  handleCheckboxUpdate,
+  checkboxLabel,
+
   ...restOfProps
 }) => {
   const textFieldRef = useRef()
 
   useStopInputScrollingIncrementNumber(textFieldRef)
 
   const [isHelperTextShowing, setIsHelperTextShowing] = useState(false)
+  const [isChecked, setIsChecked] = useState(false)
```

Comment:
Since this is a generic input component (not specific to checkboxes), we could name this state to be `isCheckboxChecked`?

Reply:
Code philosophy cheap ideas, not a change request: I wonder if another approach is just to wrap this existing input component and add the checkbox to the wrapper (if the checkbox isnt nested). If it is nested, maybe we modify this one to have a slot for the wrapper to inject the checkbox. Thinking in terms of avoiding hasty abstractions, and extension over modification.

Reply:
well update the var name ! I'm thinking of adding a separate tasks to make checkboxes more consistent across the site. I noticed we are using different styles, and wrappers.